packages/renderer/package.json

```diff
@@ -18,12 +18,15 @@
   },
   "author": "",
   "license": "AGPL-3.0-or-later",
+
+  "//": "Mermaid is bundled by buildAssets.js -- using it as an unbundled dependency significantly increases app size.",
```

Comment:
I'm not sure about leaving comments in `package.json` with `"//"` ([the suggestion in the upstream `npm` issue](https://github.com/npm/npm/issues/4482#issuecomment-32267045)). Perhaps it would be better to document this somewhere else?

Reply:
Yes i guess this is good enough. It's indeed tricky to document package.josn because of this limitation on comments